access/src/main/java/zone/cogni/asquare/virtuoso/VirtuosoRdfStoreService.java

```diff
@@ -60,20 +49,18 @@ public VirtuosoRdfStoreService(String rdfStoreUrl, String rdfStoreUser, String r
     this.rdfStoreUser = rdfStoreUser;
     this.rdfStorePassword = rdfStorePassword;
     this.graphCrudUseBasicAuth = graphCrudUseBasicAuth;
+    //  TODO HttpClientBuilder httpClientBuilder = HttpClients.custom().useSystemProperties();
+    httpClient = Utils.createHttpClientBuilder(rdfStoreUser, rdfStorePassword).connectTimeout(
```

Comment:
seems we set password here and also in httpClientBuilder method. it might be that this is how it should be done. but if is how it should be done: then please add a comment

Reply:
Good catch - probably some relict from refactoring ... fixed.